tests/acceptance/cli/test_deploy_cli.py

```diff
@@ -45,74 +54,6 @@ def test_echo_solidity_version(click_runner):
     assert str(SOLIDITY_COMPILER_VERSION) in result.output, 'Solidity version text was not produced.'
 
 
-def test_nucypher_deploy_contracts(click_runner,
```

Comment:
Note for reviewers: Here we are beginning the retirement + deprecation of the automated series deployment for *all* contracts.  Single contract deployment is the new recommended way.